demo/style.css

```
@@ -0,0 +1,362 @@
+  font-family:
+    Inter,
+    ui-sans-serif,
+    system-ui,
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    sans-serif;
+  color: #e7edf4;
+  background: #071019;
+  font-synthesis: none;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.14.1)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)